src/client/components/Login.js

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
 
-const Login = () => <h1>Login or Sign up</h1>;
+const Login = () => (
```

Comment:
Probably would just want this component to have links to `/auth/google` and `/auth/facebook`, right?